.github/workflows/uploadFirtoolBinaries.yml

```diff
@@ -128,21 +129,21 @@ jobs:
       - name: Package Binaries
         run: |
           mv install ${{ steps.name_dir.outputs.value }}
-          ${{ matrix.tar }} czf circt-bin-${{ matrix.runner }}.tar.gz ${{ steps.name_dir.outputs.value }}
+          ${{ matrix.tar }} czf firtool-bin-${{ matrix.runner }}.tar.gz ${{ steps.name_dir.outputs.value }}
       - name: Show Tarball
         run: |
-          ls -l circt-bin-${{ matrix.runner }}.tar.gz
-          shasum -a 256 circt-bin-${{ matrix.runner }}.tar.gz
+          ls -l firtool-bin-${{ matrix.runner }}.tar.gz
+          shasum -a 256 firtool-bin-${{ matrix.runner }}.tar.gz
       - name: Upload Binaries (Non-Tag)
         uses: actions/upload-artifact@v3
         if: github.ref_type != 'tag'
         with:
-          name: circt-bin-${{ matrix.runner }}.tar.gz
-          path: circt-bin-${{ matrix.runner }}.tar.gz
+          name: firtool-bin-${{ matrix.runner }}.tar.gz
+          path: firtool-bin-${{ matrix.runner }}.tar.gz
           retention-days: 7
       - name: Upload Binaries (Tag)
         uses: AButler/upload-release-assets@v2.0
         if: startsWith(github.ref, 'refs/tags/')
         with:
-          files: circt-bin-${{ matrix.runner }}.tar.gz
+          files: firtool-bin-${{ matrix.runner }}.tar.gz
```

Comment:
How do people feel about this artifact name change? IMO it more accurately captures what this artifact is (with this change), but it will break anyone relying on the artifact name in automation, eg. https://github.com/chipsalliance/chisel/blob/5be1fbc0e319ffc939b2cb64fb460e1b416d7e64/.github/workflows/install-circt/action.yml.

My thinking is that if we decide to distribute larger circt releases (including on firtool tags), that can then reclaim the `circt-bin-*` artifact name.

Reply:
Seems about right as we could add a full-circt release as well if we needed it

Reply:
I might go with firrtl-bin based on the use case rather than the executable name.

Reply:
Good point, I'll switch things to `firrtl`.